shared/src/domain-logic/determineForeignBusinessType.ts

```diff
@@ -0,0 +1,19 @@
+import { ForeignBusinessTypeId } from "../profileData";
```

Comment:
[sand] Change the import so it isn't a relative import.

Reply:
punting - https://www.pivotaltracker.com/story/show/186365752